portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,6 +53,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/index.html");
   }
 
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("UserComments").addSort("timestamp", SortDirection.DESCENDING);
+    List<Comments> comments = new ArrayList<>();
```

Comment:
As mentioned in the previous PR, this can be moved into the readComments method.

Reply:
Done (will be shown in next PR)